src/libs/ReportUtils.js

```diff
@@ -966,7 +966,9 @@ function buildOptimisticWorkspaceChats(policyID, policyName) {
 function isUnread(report) {
     const lastReadSequenceNumber = report.lastReadSequenceNumber || 0;
     const maxSequenceNumber = report.maxSequenceNumber || 0;
-    return lastReadSequenceNumber < maxSequenceNumber;
+
+    // maxSequenceNumber is actually reportActionsCount
```

Comment:
Discussion here: https://expensify.slack.com/archives/C01GTK53T8Q/p1670425143994779?thread_ts=1670254686.964499&cid=C01GTK53T8Q

Reply:
Putting this PR on hold until https://github.com/Expensify/Expensify/issues/240916 is done, `Rebuild "unread reports" without sequenceNumbers` which will fix this problem without this diff.